modules/client-configuration/pages/salt-client-proxy.adoc

```diff
@@ -18,26 +18,26 @@ By default, the username is [systemitem]``root``.
 . By default, the [guimenu]``Disable SSH Strict Key Host Checking`` checkbox is selected.
 This allows the bootstrap process to automatically accept SSH host keys without requiring you to manually authenticate.
 . OPTIONAL: Check the [guimenu]``Manage System Completely via SSH`` checkbox.
-If you check this option, the client will be configured to use SSH for its connection to the Server, and no other connection method will be configured.
+If you check this option, the client will be configured to use SSH for its connection to the server, and no other connection method will be configured.
 . Click btn:[Bootstrap] to begin registration.
 When the bootstrap process has completed, your client will be listed at menu:Systems[System List].
 
 
 
 // TODO: Might need an 'unsupported' note? LKB 2019-05-01
 
-Instead of the {webui}, you may use the command line to register a client through a proxy.
-This procedure requires that you have installed the Salt package on the client before registration, and have the Advanced systems module activated.
+Instead of the {webui}, you can use the command line to register a minion through a proxy.
```

Comment:
Better avoid the term "minion", and use "Salt client" instead.

Reply:
missed those. Thanks!